packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,99 @@ import './page.css';
 
 ```ts file="./PageWithOrWithoutFill.tsx"
 ```
 
 ### Main section padding
 
-```ts file="./PageMainSectionPadding.tsx"
-```
+To adjust the padding of a `<PageSection>`, you can pass in different values to the `padding` property. These values should be aligned to a specific breakpoint: 'default', 'sm', 'md', 'lg', 'xl', and '2xl'. Each breakpoint passed into the property should be given a value of either ‘padding’ or ‘noPadding’. 
 
-### Uncontrolled nav
+As the page's viewport width increases, breakpoints inherit the padding behavior of the previous breakpoint. For example, padding that is set on 'lg' also applies to 'xl' and '2xl'. 
 
-```ts file="./PageUncontrolledNav.tsx"
+To remove padding entirely, pass 'noPadding' to the `default` breakpoint. For example, the second section in this example passes in `padding={{ default: 'noPadding' }}`. Since no specific breakpoints are mentioned, every breakpoint will have 'noPadding'.
+
+To add padding at specific breakpoints, pass in "padding" at those breakpoints. For example, the third section in this example passes in `padding={{ default: 'noPadding', md: 'padding', lg: 'padding' }}`. At 'md', 'lg',  'xl' and '2xl' breakpoints, the default value will be overwritten, and padding will be added.
+
+To remove padding at specific breakpoints, pass in 'noPadding' at those breakpoints. For example, the fourth section in this example passes in `padding={{ md: 'noPadding' }}`, which means that 'md', ‘lg’  'xl' and '2xl' breakpoints will have ‘noPadding’.
+
+```ts file="./PageMainSectionPadding.tsx"
 ```
 
 ### Group section
 
-```ts file="./PageGroupSection.tsx"
-```
+To group page content sections, add one or more `<PageGroup>` components to a `<Page>`. 
 
-### Vertical nav using PageHeader component
+The following example adds a group containing `<PageNavigation>`, `<PageBreadcrumb>`, and `<PageSection>` components.
 
-This example is provided because PageHeader and PageHeaderTools are still in use; however, going forward Masthead and Toolbar should be used to make headers rather than PageHeader and PageHeaderTools.
+To add additional components and information to a group, you may use the following properties: 
 
-```ts file="./PageVerticalNavUsingPageHeaderComponent.tsx"
+- To indicate that a breadcrumb should be in a group, use `isBreadcrumbGrouped`. 
+- To indicate that tertiary navigation should be in a group, use `isTertiaryNavGrouped`.
+- To specify additional group content, use `additionalGroupedContent`. 
+
+
+```ts file="./PageGroupSection.tsx"
 ```
 
 ### Centered section
 
+By default, a page section spans the width of the page. To reduce the width of a section, use the `isWidthLimited` property. To center align width-limited page sections, use the `isCenterAligned` property. When a width-limited page section is wider than the value of `--pf-c-page--section--m-limit-width--MaxWidth`, the section will automatically be centered.
```

Comment:
> When a width-limited page section is wider than the value of `--pf-c-page--section--m-limit-width--MaxWidth`

I think folks might get what that's say, but it's a bit confusing because the section will never grow wider than that value. It's more true that when the page's main content area is wider than that value, the section stops growing in width as the main content area continues to grow. And if `isCenterAligned` is applied, that section will also be centered instead of left-aligned.

```suggestion
By default, a page section spans the width of the page. To reduce the width of a section, use the `isWidthLimited` property. To center align width-limited page sections, use the `isCenterAligned` property. When the main content area of a page is wider than the value of a centered, width-limited page section's `--pf-c-page--section--m-limit-width--MaxWidth` custom property, the section will automatically be centered.
```